widgets/features/features.php

```diff
@@ -220,6 +220,20 @@ function get_widget_form(){
 				'default' => false,
 			),
 
+			'title_tag' => array(
+				'type' => 'select',
+				'label' => __( 'Title Text HTML Tag', 'so-widgets-bundle' ),
```

Comment:
@AlexGStapleton SO is using Sentence case so `Title Text HTML Tag` should be changed to `Title text HTML tag`. Personally, I'd prefer Title Case but to match the standard already in use across the plugin and Widgets Bundle, it should be sentence case.

Reply:
Ah, sorry I had forgotten to check this.

Reply:
No worries, all sorted now.